src/Models/QSM/qsm_sb.m

```diff
@@ -110,7 +110,7 @@
 
 properties (Hidden = true)
 
-onlineData_url = getLink;
+onlineData_url = getLink_qsm();
```

Comment:
When did this feature get changed? Do we write different function names (eg. getLink_VFA(), getLink_inversion_recovery()) for each model? Why not just getLink?